### DISCUSSION TITLE
inviteToChannel

### DISCUSSION THREAD

**Discussion by unsafePtr** [egramtel/tdsharp#122] [Q&A] [answered]
Hi,

Am I right that there are no all functions from the TdLib? For example I've tried to find [inviteToChannel](https://core.telegram.org/method/channels.inviteToChannel) but seems there is no such within the codebase.

**Comment by x2bool** [comment, marked as answer]
Hi! TdLib API is not the same as Telegram API. They are similar, but not 1:1.

https://github.com/tdlib/td/blob/master/td/generate/scheme/td_api.tl - you have to look at this file to find the method you need.
